types/ramda/index.d.ts

```diff
@@ -1232,7 +1232,7 @@ export function nth(n: number): <T>(list: readonly T[]) => T | undefined;
 /**
  * Returns a function which returns its nth argument.
  */
-export function nthArg(n: number): (...a: readonly any[]) => any;
+export function nthArg<T>(n: number): (...a: readonly any[]) => T;
```

Comment:
Dont see what the benefit of this.. 
You always have to specify the type and that can be specified as :
`R.nthArg(1)(1, 'b', false) as string` instead

Reply:
Have a look on my comment bellow: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/44768#issuecomment-632019919